.travis.yml

```diff
@@ -12,10 +12,11 @@ matrix:
 #       env: PUPPET_GEM_VERSION="~> 3.7.5"
 #     - rvm: 1.8.7
 #       env: PUPPET_GEM_VERSION="~> 3.8.1"
-    - rvm: 1.9.3
-      env: PUPPET_GEM_VERSION="~> 3.8.1"  FUTURE_PARSER="yes"
-    - rvm: 2.0.0
-      env: PUPPET_GEM_VERSION="~> 3.8.1"  FUTURE_PARSER="yes"
+# Disabling Puppet < 4 as no longer supported
+#    - rvm: 1.9.3
+#      env: PUPPET_GEM_VERSION="~> 3.8.1"  FUTURE_PARSER="yes"
+#    - rvm: 2.0.0
```

Comment:
@MikaelSmith have updated this now and travis is happy. Let me know if you need any more changes to this.